src/components/teacher/reading.cjsx

```diff
@@ -0,0 +1,84 @@
+React = require 'react'
+BS = require 'react-bootstrap'
+Router = require 'react-router'
+Widgets = require 'react-widgets'
+Datepicker = Widgets.DateTimePicker
+
+SelectReadings = React.createClass
+  mixins: [BS.OverlayMixin],
+  getInitialState: -> {
+    isModalOpen: false
+  }
+
+  handleToggle: ->
+    @setState({
+      isModalOpen: !@state.isModalOpen
+    })
+
+  render: ->
+    <BS.Button onClick={@handleToggle} bsStyle="primary">Edit Readings</BS.Button>
+
+  renderOverlay: ->
+    if !@state.isModalOpen
+      return <span/>
```

Comment:
No, looks like we need to return something there in order for this to work.  Just the way the react-bootstrap modal component works.